### DISCUSSION TITLE
[Projects board] Notification when moving a card / issue between two columns

### DISCUSSION THREAD

**Discussion by philippart-s** [community/community#23915] [New to GitHub]
<p>Hi,</p>
<p>Is there a way to have a notification when a card or an issue is moved in a columnn in the projects board ?</p>
<p>For example to be alerted when I do test some functionalities I want ot receive a notification when my card / issue is moved in a particular column.</p>
<p>Regards,</p>
<p>Stephane</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/philippart-s">@philippart-s</a>!</p>
<p>Currently there is not a way to get notifications for this. However, I have passed on your suggestion to our product team and it will be announced in the <a href="https://github.blog/changelog/" rel="nofollow noopener">GitHub Changelog</a> if they implement this feature.</p>
<p>Thanks!</p>

**Reply by liesislukas** [reply]
wonder how many more years till this would be done? how to manage a team when you can't even get notification about fundamental change of the project?

**Reply by djschny** [reply]
2019 popped it's head back in here in 2024 to see if the distraction of AI has worn off yet so real work can happen.

**Reply by mishmanners** [reply]
You could also create a GitHub Action for this 😉

**Reply by dtrezubSV** [reply]
Banging my head to the wall for the past 2 weeks, trying to figure out how to make this work, just to find out it's not possible. 

@mishmanners there is no trigger available in projects v2 for this :(

**Reply by Myoldmopar** [reply]
Yeah, this feels like a pretty fundamental need.  I'd love to be able to trigger notifications for changes to card data in a given column or any column.  I keep missing updates from my team when they make a request, so we have to manually ping rather than just letting GitHub notify me of the change they are already making.

**Reply by dev-000** [reply]
Really sad to see this not being implemented in 5years. It's a basic feature when folks would want to get notified via Github or some third party integrations in Teams when there is a change of card from one column to another.

**Comment by philippart-s** [comment]
<p>ok thanks for your answer.</p>

**Comment by vandanamal** [comment]
<p>Was this ever implemented?</p>

**Comment by lanepemberton** [comment]
<p>Status update? Surprised this isn’t already a feature</p>

**Comment by Martijncvv** [comment]
Sir, wen feature

**Comment by liesislukas** [comment]
still nothing? what a bummer...

**Comment by dozzzzer** [comment]
Please consider adding this, it's of high importance for me and my team. Thank you.
@ThirdEyeRose

**Comment by BenjaminSIP** [comment]
Please add this feature

**Comment by JanSher076** [comment]
I want a notification if a card is moved to 'Testing' column.

**Comment by iustin94** [comment]
4 years and not implemented? What?

**Comment by 3Dr00N3Y** [comment]
2024 reporting in to say "wow".

**Comment by zecoryx** [comment]
Response #4

**Comment by werlior** [comment]
Aw man really, this still isn't possible?

**Comment by MrTinker64** [comment]
I would also love to see this implemented

**Comment by akzn** [comment]
yooo hows the update for this feature??
